model/feedback.go

```diff
@@ -0,0 +1,44 @@
+package model
+
+import (
+	"time"
+
+	"github.com/mongodb/anser/bsonutil"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/evergreen-ci/evergreen/db"
+)
+
+const FeedbackCollection = "feedback"
+
+var (
+	FeedbackTypeKey = bsonutil.MustHaveTag(FeedbackSubmission{}, "Type")
+)
+
+type FeedbackSubmission struct {
+	Type        string           `json:"type" bson:"type"`
```

Comment:
should this be an enum rather than an arbitrary string?

Reply:
I'd like to keep as a string for now since we're not sure how we want to store these going forward